docker/ssh/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2021 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
+
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+# Install openssh-server
+RUN apt update && apt install  openssh-server -y
+
+# Create privilege separation directory
+RUN mkdir -p /run/sshd
+
+# Create kernelci user
+RUN useradd kernelci -u 1000 -d /home/kernelci -s /bin/bash
+RUN mkdir -p /home/kernelci
+RUN chown kernelci: /home/kernelci
+
+# SSH server will run on PORT 22
+EXPOSE 22
+
+# Run SSH server in detached mode
+CMD ["/usr/sbin/sshd","-D"]
```

Comment:
Note: The red sign at the end of the line is because the file is missing a final new line.  It's a UNIX thing and Git will also complain about it.  Not a big issue, if you don't mind pushing to the branch once again then it would be a bit cleaner to resolve it.

Here's a related discussion about this topic:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Committed the changes. @gctucker